docs/guides/getting-started-java-spring.md

```diff
@@ -1,15 +1,22 @@
 ---
 id: getting-started-java-spring
-title: Getting started as a Java developer using Spring
-sidebar_label: Getting started as a Java developer using Spring
+title: Get started as a Java developer using Spring
```

Comment:
I'll have another look at these naming conventions in https://github.com/camunda/developer-experience/issues/253.